php/one-fpm/App/index.php

```diff
@@ -4,6 +4,11 @@
 
 define('_APP_PATH_VIEW_', __DIR__ . '/View');
 
+function uuid()
```

Comment:
what is that function for ?

Reply:
By default, the framework sets a unique id for logging. There is no need for logging here. I rewritten it.